gradio/analytics.py

```diff
@@ -120,10 +120,10 @@ def launched_analytics(blocks: gradio.Blocks, data: dict[str, Any]) -> None:
             if y in blocks.blocks
         ]
         inputs_telemetry = inputs_telemetry + [
-            str(blocks.blocks[y]) for y in x["inputs"]
+            str(blocks.blocks[y]) for y in x["inputs"] if y in blocks.blocks
```

Comment:
If `y` if not in `blocks.blocks` shouldn't we also raise an InvalidBlocksError here?

Reply:
I don't believe so. That would prevent the blocks from launching, which is what we should allow IMO because someone could be iteratively developing -- we should let the blocks launch but only raise an error if they try to use the event that is invalid

Reply:
Should we not launch telemetry then? I'm worried the data would be nonsensical if the block is invalid

Reply:
Agreed, that's why we don't include the block in the list of blocks if it's invalid (i.e. we only include `blocks.blocks[y]` if `y in blocks.blocks`)